### DISCUSSION TITLE
can't import experimental components from @wordpress/components

### DISCUSSION THREAD

**Discussion by ivanjeremic** [WordPress/gutenberg#38213] [General]
I try to use Navigator sceens https://developer.wordpress.org/block-editor/reference-guides/components/navigator/ but trying to use it shows me that `@wordpress/components` has no exports for it, I searched and found that I should do 

```
import {
	__experimentalNavigator as Navigator,
	__experimentalNavigatorScreen as NavigatorScreen,
	__experimentalUseNavigator as useNavigator,
} from "@wordpress/components";
``` 
but this doesn't work. The component package version I use is `^19.2.0`

**Comment by talldan** [comment]
Experimental components aren't really intended for public use, you can use them at your own risk. But they might be changed, renamed, or removed entirely without any notice.

I would check you do actually have 19.2 installed in the `node_modules` folder. Node dependencies aren't always straightforward. There have been other reports of this kind of thing (https://github.com/WordPress/gutenberg/issues/36693), so it probably indicates another package is outdated and requiring an older version of components or something like that, and you might need to update that package.

I don't think there's any reason importing wouldn't work if you have the right version.
